lib/rspec_log.rb

```diff
@@ -8,8 +8,8 @@ class RSpecLog
   def initialize(filename: DEFAULT_LOG_FILE, newfile: false)
     raise 'RSpec must be defined to create RSpec log' if (defined? RSpec).nil?
 
-    @filename = filename
-    RSpecLog.write_hash_to_file({}, @filename) if newfile || !File.exist?(filename)
+    @filename = filename || DEFAULT_LOG_FILE
```

Comment:
It is, but if you explicitly pass `nil`, it becomes `nil`. The default is there only for when it is not specified.

A simple example:

```ruby
def foo(bar: 'baz')
  puts bar
end

foo
=> baz # prints baz
 
foo(bar: nil)
=>  # prints empty string

```